src/cljdoc/server/sitemap.clj

```diff
@@ -0,0 +1,30 @@
+(ns cljdoc.server.sitemap
+  (:require [sitemap.core :refer [generate-sitemap]]
+            [clojure.tools.logging :as log]
+            [cljdoc.config :as cfg]
+            [cljdoc.storage.api :as storage]
+            [cljdoc.server.routes :as routes]))
+
+
+(defn sitemap [db-spec]
+  (generate-sitemap
+   (for [{:keys [group_id artifact_id] :as doc} (storage/all-distinct-docs db-spec)]
+     {:loc
+      (routes/url-for :artifact/current :path-params {:group-id group_id
```

Comment:
We want this URL to be the current versioned URL, not the root URL which will redirect to the versioned URL.